src/main/java/securbank/models/Otp.java

```diff
@@ -0,0 +1,153 @@
+package securbank.models;
+
+import java.util.UUID;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import javax.validation.constraints.NotNull;
+import org.hibernate.annotations.GenericGenerator;
+import org.joda.time.LocalDateTime;
+
+
+/**
+ * @author Joel Mascarenhas
+ *
+ */
+
+@Entity
+@Table(name = "Otp")
+public class Otp {
+
+	/**
+	 * @param userId
+	 * @param OTPId
+	 * @param code
+	 * @param active
+	 * @param createdOn
+	 */
+
+	public Otp(UUID userId, Integer OTPId, Integer code, Boolean active,LocalDateTime createdOn) {
+		super();
+		this.userId = userId;
+		this.OTPId = OTPId;
+		this.code = code;
+		this.active = active;
+		this.createdOn = createdOn;
+	}
+
+	@Id
+	@GeneratedValue(generator = "uuid2")
+	@GenericGenerator(name = "uuid2", strategy = "uuid2")
+	@NotNull
+	@Column(name = "OTPId", unique = true, nullable = false, columnDefinition = "BINARY(16)")
+	private Integer OTPId;
+	
+	@NotNull
+	@Column(name = "code", nullable = false, updatable = false)
+	private Integer code;
+	
+	@Column(name = "userId", unique = true, nullable = false, columnDefinition = "BINARY(16)")
```

Comment:
We need to create a relationship between user and dao.
Please look for foreign keys in hibernate or jpa.
You will have to create `ManyToOne` and `OneToMany` relationship